### PR TITLE
feat(Aodesu): add presence

### PR DESCRIPTION
**Pull request by DevAoDesu**
## Description 

Aodesu is a website where you can watch anime in spanish dub or subtitled in spanish.

## Acknowledgements
- [x] I read the [Presence Guidelines](https://github.com/PreMiD/Presences/blob/main/.github/CONTRIBUTING.md)
- [x] I linted the code by running `npm format`
- [x] The PR title follows the repo's [commit conventions](https://github.com/PreMiD/Presences/blob/main/.github/COMMIT_CONVENTION.md)

## Screenshots
<details>
<summary> Proof showing the creation/modification is working as expected </summary>
<h3>Viewing Home</h3>

<hr/>

<h3>Viewing Emitting</h3>

<hr/>

<h3>Viewing Anime Landing</h3>

<hr/>

<h3>Viewing Reproductor Playing</h3>

<hr/>

<h3>Viewing Reproductor Pause</h3>

<hr/>

<h3>Viewing Search</h3>

<hr/>

<h3>Viewing Settings > Appearance</h3>

<hr/>

<h3>Viewing Settings > Privacy</h3>

<hr/>

<h3>Viewing Register</h3>

<hr/>

<h3>Viewing Login</h3>

<hr/>

</details>

### CODE REVIEW

File: websites/A/Aodesu/presence.ts [hunk 1]
```diff
@@ -0,0 +1,154 @@
...
+			{
+				label: "¡Ver También!",
+				url: href,
+			},
+		];
+	} else if (pathname.startsWith("/buscar")) {
+		const search = document.querySelector<HTMLInputElement>(
+			"div.search-bar > input[type=text]"
+		).value;
+		presenceData.smallImageKey = "search";
```
Comment (theusaf) [inline websites/A/Aodesu/presence.ts:105]:
```suggestion
		presenceData.smallImageKey = Assets.Search;
```

File: websites/A/Aodesu/presence.ts [hunk 2]
```diff
@@ -0,0 +1,154 @@
...
+
+presence.on("UpdateData", async () => {
+	const presenceData: PresenceData = {
+			largeImageKey: Assets.Logo,
+			startTimestamp: browsingTimestamp,
+		},
+		{ href, pathname } = document.location;
+
+	if (pathname === "/") {
+		// Home Page
```
Comment (theusaf) [inline websites/A/Aodesu/presence.ts:53]: These comments are mostly redundant, obvious from looking at the code, so they should be removed.

**Review by theusaf — CHANGES_REQUESTED** [review]

File: websites/A/Aodesu/presence.ts (hunk 2 above)
Comment (DevAoDesu) [reply websites/A/Aodesu/presence.ts:53]: I remove it

**Review by theusaf — APPROVED** [review]
LGTM

File: websites/A/Aodesu/presence.ts [hunk 3]
```diff
@@ -0,0 +1,146 @@
...
+			},
+		];
+	} else if (pathname.startsWith("/buscar")) {
+		const search = document.querySelector<HTMLInputElement>(
+			"div.search-bar > input[type=text]"
+		).value;
+		presenceData.smallImageKey = Assets.Search;
+		presenceData.smallImageText = "Buscador";
+		presenceData.details = "Buscando";
+		presenceData.state = search;
```
Comment (DarkVillager) [inline websites/A/Aodesu/presence.ts:100]:
```suggestion
		presenceData.smallImageKey = Assets.Search;
		presenceData.smallImageText = "Buscador";
		presenceData.details = "Buscando";
		presenceData.state = document.querySelector<HTMLInputElement>("div.search-bar > input[type=text]").value;
```

File: websites/A/Aodesu/presence.ts [hunk 4]
```diff
@@ -0,0 +1,146 @@
...
+			presenceData.details = "¡Creando nueva cuenta!";
+		}
+	} else if (pathname.startsWith("/serie/")) {
+		const title = document.querySelector("h1"),
+			season = document.querySelector("#season_id");
+
+		presenceData.smallImageKey = Assets.Preview;
+		presenceData.smallImageText = "Serie";
+		presenceData.details = `A punto de ver ${title.textContent}`;
+		presenceData.state = `Temporada ${season.textContent}`;
```
Comment (DarkVillager) [inline websites/A/Aodesu/presence.ts:86]:
```suggestion
		presenceData.smallImageKey = Assets.Preview;
		presenceData.smallImageText = "Serie";
		presenceData.details = `A punto de ver ${document.querySelector("h1").textContent}`;
		presenceData.state = `Temporada ${document.querySelector("#season_id").textContent}`;
```

**Review by DarkVillager — COMMENTED** [review]
Let's remove variables only used once.

File: websites/A/Aodesu/presence.ts (hunk 3 above)
Comment (DarkVillager) [inline websites/A/Aodesu/presence.ts:100]:
```suggestion
		presenceData.smallImageKey = Assets.Search;
		presenceData.smallImageText = "Buscador";
		presenceData.details = "Buscando";
		presenceData.state = document.querySelector<HTMLInputElement>("div.search-bar > input[type=text]").value;
```

File: websites/A/Aodesu/presence.ts (hunk 4 above)
Comment (DarkVillager) [inline websites/A/Aodesu/presence.ts:86]:
```suggestion
		presenceData.smallImageKey = Assets.Preview;
		presenceData.smallImageText = "Serie";
		presenceData.details = `A punto de ver ${document.querySelector("h1").textContent}`;
		presenceData.state = `Temporada ${document.querySelector("#season_id").textContent}`;
```

**Review by DarkVillager — CHANGES_REQUESTED** [review]

**Review by theusaf — APPROVED** [review]

**Review by DarkVillager — APPROVED** [review]

**Review by Bas950 — APPROVED** [review]

**Pull request merged**